<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>29</y>
      <width>1111</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>591</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="detectButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>DETECT FACE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="grayButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>GRAY MODE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startRecordButton">
       <property name="minimumSize">
        <size>
         <width>89</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>START RECORD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>SAVE </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopRecordButton">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>STOP RECORD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
